dashboard/src/app/projects/create-project/create-project.controller.ts

```diff
@@ -1067,14 +973,14 @@ export class CreateProjectController {
       // generate a name
 
       // starts with project
-      var name = 'project';
+      let name = 'project';
```

Comment:
Here `name` is inferred to be `string` because of its value.
[Type Inference](https://www.typescriptlang.org/docs/handbook/type-inference.html)

Should we specify the type of the variable in this case?